File: ADSync/OS2faktorADSync/Model/Kombit/Jfr.cs
using Newtonsoft.Json;

namespace OS2faktorADSync.Model
{
    public class Jfr
    {
        [JsonProperty(PropertyName = "identifier")]
        public string Identifier { get; set; }

        [JsonProperty(PropertyName = "cvr")]
        public string CVR { get; set; }
    }
}

File: ui/src/main/java/dk/digitalidentity/api/dto/Jfr.java
package dk.digitalidentity.api.dto;

import dk.digitalidentity.common.dao.model.KombitJfr;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@EqualsAndHashCode
public class Jfr {
	private String identifier;
	private String cvr;

	public Jfr(KombitJfr jfr) {
		this.identifier = jfr.getIdentifier();
		this.cvr = jfr.getCvr();
	}
}
